src/main/java/com/minecolonies/coremod/commands/ColonyCommand.java

```diff
@@ -2,6 +2,8 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.minecolonies.coremod.commands.colonycommands.*;
+import com.minecolonies.coremod.commands.colonycommands.requestsystem.RSCommand;
+import com.minecolonies.coremod.commands.colonycommands.requestsystem.RSResetCommand;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.commands.colonycommands.requestsystem.RSResetCommand'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)